index.js

```diff
@@ -936,11 +1155,32 @@ function importConverterGeneral(path, context) /*: K.Import */ {
       };
     }
 
+      exported = exportAll[0];
```

Comment:
Currently this is an educated guess that there will only ever be one exportAll in a file. What's the contingency for multiple exportAlls? This looks like it will create unpredictable missing type information when it gets the wrong exportAll.

The easiest option would be to throw an error here if multiple exportAlls are found, or otherwise break and error.